examples/local-access/.saturn/saturn.json

```diff
@@ -0,0 +1,22 @@
+{
+    "image": "saturncloud/examples-gpu:2021.02.22",
+    "jupyter": {
+        "size": "g4dnxlarge",
+        "disk_space": "10Gi",
+        "ssh_enabled": false
+    },
+    "dask_cluster": {
+        "n_workers": 3,
+        "nthreads": 1,
+        "scheduler_size": "medium",
```

Comment:
i think this needs to be large for freemium

Reply:
I'll check and see what runs, I didn't test it and should've!